nix/modules/tests/kld-mgr.nix

```diff
@@ -108,6 +109,12 @@ in
       installer.wait_until_succeeds("ssh -o StrictHostKeyChecking=no root@192.168.42.2 -- exit 0 >&2")
 
       new_machine.wait_for_unit("sshd.service")
+
+      system_info = installer.succeed("${lib.getExe kld-mgr} --config  /root/test-config.toml system-info --hosts kld-00").strip()
+      assert system_info.startswith("[kld-00]\nkld-version: 0.1.0\ngit-sha:"), f"unexpected system info: {system_info}"
+
+      system_info = installer.succeed("${lib.getExe kld-mgr} --config  /root/test-config.toml system-info --hosts db-00").strip()
+      assert system_info.startswith("[db-00]\nkld-version: 0.1.0\ngit-sha:"), f"unexpected system info: {system_info}"
```

Comment:
Is there some way you can add checks for the bitcoind and cockroach version? Not specific versions, just a check that some version information is returned for those services on the hosts that they should be deployed on.